web/web.go

```diff
@@ -770,26 +770,12 @@ func dropSecondaryIndex(w http.ResponseWriter, r *http.Request) {
 }
 
 // updateSessionFile updates the content of session file with
-// latest sessionState.conv.
+// latest sessionState.conv while also dumping schemas and report.
 func updateSessionFile() error {
-	filePath := sessionState.sessionFile
-	if filePath == "" {
-		return fmt.Errorf("Session file path is empty")
-	}
-	f, err := os.OpenFile(filePath, os.O_RDWR|os.O_CREATE, 0644)
-	if err != nil {
-		return err
-	}
```

Comment:
Do we know why was this code required before, and why is it okay to drop it now?

Reply:
Not sure of the pros of opening, clearing then writing compared to simply doing a write. But I did verify the new changes and overwriting does the job.

Reply:
Sounds good. Thanks for verifying.